doc/rtd/topics/datasources/vmware.rst

```diff
@@ -7,13 +7,88 @@ This datasource is for use with systems running on a VMware platform such as
 vSphere and currently supports the following data transports:
 
 
+* `Guest OS Customization <https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-58E346FF-83AE-42B8-BE58-253641D257BC.html>`_
 * `GuestInfo <https://github.com/vmware/govmomi/blob/master/govc/USAGE.md>`_ keys
 
 Configuration
 -------------
 
 The configuration method is dependent upon the transport:
 
+Guest OS Customization
+^^^^^^^^^^^^^^^^^^^^^^
+
+The following configuration can be set for this datasource in cloud-init
+configuration (in `/etc/cloud/cloud.cfg` or `/etc/cloud/cloud.cfg.d/`).
+
+System configuration:
+
+``disable_vmware_customization``:
+true(disable) or false(enable) the vmware customization. (default: true)
```

Comment:
You mentioned "traditional guest customization" in line 59, we'd better to call this  as "traditional guest customization" too in this line.

Reply:
We can explain a little bit about what is "traditional guest customization" if possible.

Reply:
Thanks for the review. Right, I shall explain the "traditional guest customization", and also mention 'disable_vmware_customization: false' is required to make sure this datasource can be found in ds-identify when using guest os customization transport.